src/cljs/rems/spa.cljs

```diff
@@ -263,7 +263,10 @@
   (rf/dispatch [:set-active-page :administration]))
 
 (secretary/defroute "/create-catalogue-item" []
-  (rf/dispatch [:rems.administration.catalogue/reset-create-catalogue-item])
+  (rf/dispatch [:rems.administration.catalogue-item/start-fetch-workflows])
+  (rf/dispatch [:rems.administration.catalogue-item/start-fetch-resources])
+  (rf/dispatch [:rems.administration.catalogue-item/start-fetch-forms])
+  (rf/dispatch [:rems.administration.catalogue-item/reset-create-catalogue-item])
```

Comment:
I'm not sure I like this pattern (even though I've used it myself). Is there some way we could keep the knowledge of which stuff needs to be started/resetted within the rems.administration.catalogue-item namespace?

How about having some sort of `:rems.administration.catalogue-item/enter-page` event that gets dispatched here, and which then causes these other events?

Reply:
I was thinking about the same thing; an enter-page event. Shall I change all pages to use that style? I’ll make it in another PR because it affects so many files.